back-end/sports_platform/sports_platform_api/views/user_views.py

```diff
@@ -25,7 +46,8 @@ def create_user(request):
         guest.register(db_data)
     except ValueError:
         return Response(data = {"message": 'There is an error regarding the provided data'}, status=400)
-    except IntegrityError:
+    except IntegrityError as e:
+        print(e)
```

Comment:
We can delete the print statement if not needed.

Reply:
sure, sorry for this.